### ISSUE TITLE
Time required for the training of current pretrained models 

### ISSUE THREAD

**Issue by usamireko** [KdaiP/StableTTS#10]
Hi

I discovered this project and is pretty amazing the results provided, I saw that it got updated to get support for the Japanese language, and that gave me the curiosity of how many epochs or hours of training were required for the current pretrained models provided, so if is possible to know, I'll be very thankful since I'm gonna train Japanese from scratch and I would like to have some idea of the amount of time/hardware I'll require 

Thanks!

**Comment by KdaiP** [comment]
Hi, the latest model is trained on 3090 for 20 days. 

Even after several days of training, the loss of flow-matching may not seem to decrease, but the details of the speech will continue to improve.
